opengrok-web/src/main/java/org/opengrok/web/messages/MessagesContainer.java

```diff
@@ -222,6 +234,7 @@ public void setMessageLimit(int limit) {
     }
 
     public void startExpirationTimer() {
+        setMessageLimit(RuntimeEnvironment.getInstance().getMessageLimit());
```

Comment:
getMessageLimit() ? i.e. why call into RuntimeEnvironment when there is the callback ?

Reply:
To be honest, I was not thinking about this too much. This code was there before in `RuntimeEnvironment` so I wanted to preserve the functionality:
```java
    public void startExpirationTimer() {
         messagesContainer.setMessageLimit(getMessageLimit());
         messagesContainer.startExpirationTimer();
     }
```
I think I can set the limit in the constructor and leave this call here out..

Reply:
Also because this method is called when the webapp starts and it's not calling the applyConfig where the callbacks are executed